server/game/cards/19-JS/Wraith.js

```diff
@@ -0,0 +1,33 @@
+const DrawCard = require('../../drawcard');
+
+class Wraith extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            match: card => card.name === 'Dale Seaworth',
+            effect: ability.effects.addKeyword('stealth')
+        });
+
+        this.action({
+            title: 'Kneel location',
+            cost: [
+                ability.costs.kneelSelf(),
+                ability.costs.sacrificeSelf()
+            ],
+            targets: {
+                location: {
```

Comment:
This nesting (`targets.location`) is unnecessary when only selecting one set of targets. It's fine, but just `target` would be simpler.

Reply:
good catch, fixed